plugins/module_utils/dnac.py

```diff
@@ -2160,13 +2160,26 @@ def compare_list(list1, list2):
 def fn_comp_key(k, dict1, dict2):
     return dnac_compare_equality(dict1.get(k), dict2.get(k))
 
+def normalize_ipv6_address(ipv6):
+    """
+    Normalize an IPv6 address for consistent comparison.
+    """
+    try:
```

Comment:
if not isinstance(ipv6, str):
        raise TypeError("Input must be a string representing an IPv6 address.")

    try:
        normalized_address = str(ipaddress.IPv6Address(ipv6))
        return normalized_address

Reply:
modified